### PR TITLE
update https install section

### PR DESCRIPTION
**Pull request by vskubriev**
### Motivation and context
Update https install manual. Makes it easier and more robust. Includes automatic renewing of lets encrypt certificates.

### How has this been tested?
This doc update has been tested on VPS manually.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by azhavoro** [comment]
@vskubriev Hi, thanks for the contribution!

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Formats: definitions are changed, are not stored in DB anymore (https://github.com/opencv/cvat/pull/1352)
 - cvat-core: session.annotations.put() now returns identificators of added objects (https://github.com/opencv/cvat/pull/1493)
 - Images without annotations now also included in dataset/annotations export (https://github.com/opencv/cvat/issues/525)
+- Update https install manual. Makes it easier and more robust. Includes automatic renewing of lets encrypt certificates.
```
Comment (nmanovic) [inline CHANGELOG.md:26]: Could you please add the link on PR?

Comment (nmanovic) [reply CHANGELOG.md:26]: Also the file has merge conflict now (sorry for that).

Comment (vskubriev) [reply CHANGELOG.md:26]: I didn’t manage to add the link to PR in time. Sorry.

**Pull request merged**